### DISCUSSION TITLE
Opção "Salvar Arquivo"

### DISCUSSION THREAD

**Discussion by hucf** [dgadelha/Portugol-Webstudio#295] [Ideias e sugestões]
A opção "Salvar Arquivo" direciona para a pasta "Download". Tem como o usuário configurar para salvar em outra pasta ?

**Comment by dngadelha** [comment]
Olá, @hucf!

Adicionamos a opção que você sugeriu! Agora você pode escolher o local que deseja salvar o arquivo clicando em **Salvar Como** -> **Escolher um arquivo**.

Lembrando que essa nova opção só está disponível para os navegadores **Chrome** e **Edge**, pois são os únicos navegadores compatíveis com esse recurso no momento.

![image](https://github.com/user-attachments/assets/397651b7-7e3a-4302-8bf6-d14f3e722224)

**Comment by hucf** [comment]
Ola , valeu Danilo !!!

Em sáb., 4 de jan. de 2025 às 14:51, Danilo Gadelha <
***@***.***> escreveu:
